application/views/footer.php

```diff
@@ -0,0 +1,13 @@
+<footer class="footer mt-auto py-3">
+        <div class="container pb-5">
+            <hr>
+            <span class="text-muted">
+                PAV | CodeIgniter (v3.11.1) Tutorial
+            </span>
+        </div>
+    </footer>
+
+    <script src="assets/js/popper.min.js"></script>
+    <script src="assets/js/bootstrap.min.js"></script>
+  </body>
+</html>
```

Comment:
We don't need the <body> and <html> here
This partial view is only imported into view, not the view itself

We can include the view inside other views, if we implement it like this, it will be hard to debug later on
We should always have a start and end tag in a single file 
E.g: 
<body> </body>:  OK
<html></html>:  OK
</body>: not OK

Reply:
According to the document:
-The header view include <head> tag and a part of <body> tag that open <body> tag.
-The index view include <main> tag
-The footer view include <footer> tag and <body>, <html> closing tag
If you want me to improve them. Please help me how to re-organize views

Reply:
@Khiemnx Oh, I understand your way, for the purpose of practicing only, it's ok